README.md

```diff
@@ -1,4 +1,4 @@
-# drs-filer
+# DRS-filer
 
```

Comment:
Should probably add some more text here. At least a little intro and a very basic usage section. Rest we can do later.